src/utils/url.js

```diff
@@ -228,3 +231,25 @@ export function getCanonicalUrl(doc) {
   const node = doc.getRootNode().querySelector("link[rel='canonical']");
   return (node && node.href) || '';
 }
+
+const PARSED_URL = parseUrl(self.window.location.href);
+
+/**
+ * Defaults to the pages current URL
```

Comment:
These "Defaults to the pages current URL"  comments are helpful. Could you move them to the "@param {...} parsedUrl" line just to be clearer? Then could you add comments describing the overall functions? Ex: "Returns ..."